sycl/test-e2e/KernelCompiler/kernel_compiler_sycl_jit_join.cpp

```diff
@@ -0,0 +1,240 @@
+//==- kernel_compiler_sycl_jit_join.cpp --- kernel_compiler extension tests ==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: (opencl || level_zero)
+// REQUIRES: aspect-usm_shared_allocations
+
+// UNSUPPORTED: accelerator
+// UNSUPPORTED-INTENDED: while accelerator is AoT only, this cannot run there.
+
+// UNSUPPORTED: windows && arch-intel_gpu_bmg_g21
+// UNSUPPORTED-TRACKER: https://github.com/intel/llvm/issues/17255
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 1
+// RUN: %{l0_leak_check} %{run} %t.out 1
```

Comment:
What's the trailing `1` argument for here? The test is using `main()` rather than `main(int argc, char * argv[])` so guessing ithe program isn't taking any arguments.

Reply:
Good eye! Fairly sure this was just a copy-paste mistake.